### ISSUE TITLE
Wrong colors in some Audacity popup windows. 

### ISSUE THREAD

**Issue by ghost** [luigifab/human-theme#11]
![1624173536](https://user-images.githubusercontent.com/86182381/122665810-d59fbf00-d17f-11eb-890a-36484200798f.png)

The windows that pop up when you apply an effect in Audacity seem to have the wrong background color. (it's the same color in all the theme variants). 

It mostly looks like that when you add an effect, but it also affects the "generate" tools, and some other misc ones so i'm not sure how easy that is to figure out. Not a deal breaker but you know. 

Here's how it's supposed to look i'm guessing (from 'Effects > Add/Remove Plug-Ins...') 

![1624174727](https://user-images.githubusercontent.com/86182381/122666079-88245180-d181-11eb-990d-bb809c334bee.png)

love the theme.

(ArchLinux / GTK 3.24.29 / Audacity 2.4.1)

**Comment by luigifab** [comment]
You are right, WxWidgets used by Audacity use something to set background color for theses dialog, I searched but I haven't yet found the magic CSS line.

**Comment by luigifab** [comment]
[Here](https://stackoverflow.com/q/68461515/2980105) is a partial workaround:
```
window > box > widget > widget { background-color:@theme_bg; }
```
Not sure if it's good or not.

**Comment by luigifab** [comment]
Added/Fixed in v2.0.0.
